cmake/modules/FindPTex.cmake

```diff
@@ -67,6 +67,10 @@ else ()
             "${PTEX_LOCATION}/include/wdas"
             "$ENV{PTEX_LOCATION}/include"
             "$ENV{PTEX_LOCATION}/include/wdas"
+            "${PTEX_LOCATION}/include/Ptex"
+            "${PTEX_LOCATION}/include/wdas"
+            "$ENV{PTEX_LOCATION}/include/Ptex"
+            "$ENV{PTEX_LOCATION}/include/wdas"
```

Comment:
It seems like  "${PTEX_LOCATION}/include/wdas" and the ENV version are already included above on lines 67 and 69.

Reply:
Well spotted, thanks. Fixed in another commit.

Reply:
That's great, thanks so much, @charlesfleche